lcls2_producers/smd_producer.py

```diff
@@ -104,6 +109,19 @@ def define_dets(run, det_list):
                     thisROIFunc.addFunc(projectionFunc(axis=proj_ax))
                 det.addFunc(thisROIFunc)
 
+        if detname in dimgs_args:
+            dimg_func = detImageFunc(**dimgs_args[detname])
+            det.addFunc(dimg_func)
+
+            # ROI extraction
```

Comment:
@silkenelson I think i got this part from you. Do we need it? I feel it would be more natural to remove it, and use the normal nesting of DetObjectFunc if one wants to extract ROIs from the image?

Reply:
I guess so. I suspect this came out of the quick hack.